chart/epinio-installer/values.yaml

```diff
@@ -50,3 +50,11 @@ domain: "localhost.omg.howdoi.website"
 
 # Use TLS when application images will be pulled by kubernetes. Only applies to the internal registry. (default "false")
 # --force-kube-internal-registry-tls: ""
+
+containerRegistryChart: "https://github.com/epinio/epinio-helm-chart/releases/download/container-registry-0.1.0/container-registry-0.1.0.tgz"
+epinioChart: "https://github.com/epinio/epinio-helm-chart/releases/download/epinio-0.1.0/epinio-0.1.0.tgz"
```

Comment:
You'll need to consume a release that has this in: https://github.com/epinio/epinio-helm-chart/pull/20
(to fix the default namespace not having a serviceaccount)

Reply:
https://github.com/epinio/epinio-helm-chart/releases/download/epinio-0.1.21/epinio-0.1.21.tgz

Reply:
```suggestion
epinioChart: "https://github.com/epinio/epinio-helm-chart/releases/download/epinio-0.1.21/epinio-0.1.21.tgz"
```